### ISSUE TITLE
Afwall+ 3.3.0 - View log fail

### ISSUE THREAD

**Issue by TorJensendk** [ukanth/afwall#1042]
When i click on "View log" the app crashes. When i open it again it begins applying iptable rules. 

Phone: LG g6 (h870)
AFwall version: 3.3.0
Donate version: Yes

Can provide more info if needed.

**Comment by ukanth** [comment]
Yes. Please provide logcat along with firewall rules from "show rules"/ export

**Comment by TorJensendk** [comment]
Send to your uzoftinc mail from ****@tutanota.com

**Comment by Danny3** [comment]
@TorJensendk is this still happening?
Can you try again and please close the issue if it's not happening anymore.
Version 4.0.3 doesn't crash when trying to see the log.
The log file doesn't work, which has been reported, but it doesn't crash, which I think this issue is about.
